servicemesh-install/topics/tutorial-grafana.adoc

```diff
@@ -0,0 +1,61 @@
+[[grafana_tutorial]]
+= Grafana Tutorial
+
+////
+TODO
+Need a short overview of Grafana.
+////
+
+Building on the `bookinfo` tutorial, shows you how to setup and use the Istio Dashboard to monitor mesh traffic. As part of this task, you will install the Grafana Istio add-on and use the web-based interface for viewing service mesh traffic data.
+
+Prerequisites:
+
+* OpenShift Container Catalog 3.10 or higher installed.
+* {ProductName} {ProductVersion} installed.
+* `Bookinfo` demonstration application installed.  
+
+== Accessing the Grafana Dashboard
+
+. A route to access the Grafana dashboard should already exist. Query OpenShift for details of the route. Query OpenShift for details of the route:
++
+```
+  $ oc get route grafana -n istio-system
+  NAME      HOST/PORT                               PATH      SERVICES   PORT      TERMINATION   WILDCARD
+  grafana   grafana-istio-system.127.0.0.1.nip.io             grafana    http                    None
+
+```
++
+. Launch a browser and navigate to http://grafana-istio-system.127.0.0.1.nip.io.  You should see Grafana's home screen, similar to the image below:
```

Comment:
This only works in 'oc cluster up' environments, it would be better to have something like

export GRAFANA_URL=$(oc get route -n istio-system grafana -o jsonpath='{.spec.host}')
navigate to http://${GRAFANA_URL}